app/src/main/java/org/mozilla/fenix/settings/DataChoicesFragment.kt

```diff
@@ -72,7 +75,7 @@ class DataChoicesFragment : PreferenceFragmentCompat() {
 
         requirePreference<SwitchPreference>(R.string.pref_key_experimentation).apply {
             isChecked = context.settings().isExperimentationEnabled
-            isVisible = Config.channel.isReleaseOrBeta
+            isVisible = Config.channel.isDebug
```

Comment:
Is this correct? We only allow for the experimentation opt out in debug?

Reply:
Changed to use a `FeatureFlag`.